app/build.gradle

```diff
@@ -12,10 +12,11 @@ android {
 
     defaultConfig {
         applicationId "org.openobservatory.ooniprobe"
-        minSdkVersion 14
+        minSdkVersion 23
```

Comment:
@lorenzoPrimi can we avoid bumping the `minSdkVersion` that much? Ideally, leaving it at 14 would be best.